### ISSUE TITLE
Euro currency: set as suffix

### ISSUE THREAD

**Issue by orthagh** [metabase/metabase#8903]
- Your browser and the version: Chrome 70
- Your operating system: Ubuntu 17.04
- Your databases: Mysql
- Metabase version: 0.31
- Metabase hosting environment: Linux/debian stretch
- Metabase internal database: Mysql

Don't know if it's a bug or a feature but it's very disturbing.
€ currency should be set a suffix not as prefix.

**Comment by tlrobinson** [comment]
The formatting/currency settings are new in Metabase so we're still working out the kinks.

We are using [`toLocaleString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) for currency formatting, and it uses the `locale` rather than the `currency` to determine whether to suffix or prefix the currency symbol. We've hard coded the locale to `en` for various reasons, but might need to reevaluate that, or use something else to do number formatting.

AFAIK `locale` affects the following:

1. numerals type
2. currency names
3. prefix/suffix of currency symbol
4. thousand/decimal separators
5. probably other things i'm missing

**Comment by SebCorbin** [comment]
See also https://github.com/metabase/metabase/issues/10785#issuecomment-888780545

**Comment by ixipixi** [comment]
We have a comment about this issue related to Swedish krona as well. It should be a suffix rather than a prefix:
https://github.com/metabase/metabase/issues/47374#issuecomment-2717715594

**Comment by kamilmielnik** [comment]
Caused by hardcoded `en` locale (multiple places): https://github.com/metabase/metabase/blob/9668ebe98c56af08e4aa9b3a1e6ccddaa59d4435/frontend/src/metabase/lib/formatting/numbers.tsx#L189


I think the way to fix it is to add `locale` prop to every currency in `src/metabase/util/currency.cljc` and then pass that information to `Intl.NumberFormat` fe-side, so I'm adding backend label.

**Comment by SebCorbin** [comment]
I would say locale != currency, for example a french user would like to see `100 $` meanwhile an american user should see `$100`

**Comment by lacak-sk** [comment]
Yes, please set € as suffix, when there is set "Euro" as currency in global settings. Values formated as €100 seems very strange.
